portfolio/src/main/webapp/index.html

```diff
@@ -1,133 +1,143 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div class="topnav">
-        <a class="active" href="#home">Main</a>
-        <a href="#news">About</a>
-        <a href="#contact">Projects</a>
-        <a href="#news">Hobbies</a>
-        <a href="#news">Photos</a>
-        <a href="#about">Links</a>
-    </div>
-    <div id="content">
-
-        <div id="animBg"><div class="myDIV"></div></div>
-        <div id="intro">
-            <div class="profile-picture" >
-              <img src="images/img_00_00.jpg"/>
-            </div>
-            <div class="basic-info">
-              <p class="name">Natalia Denisenko</p>
-              <p class="occupation">Bachelor student Computer Science Faculty in National Research University "Higher school of Econimics".</p>
-              <p class="location">Russia, Moscow.</p>
+    <head>
+        <meta charset="UTF-8">
```

Comment:
Please use 2 spaces for indentation as per the style guide. It is usually easier if can change the default indentation used by your editor so that you don't have to manually keep doing it.

Reply:
Done.